CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+### Breaking changes
+
+- Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+  Both static mode's `remote_write` and flow mode's `prometheus.remote_write` are affected. (@wildum)
+
```

Comment:
Sorry, but now that the change for flow mode and the change for static mode are in different releases, this will need to be split into two entries again!

Reply:
oops I hope this time I got it correctly